### PR TITLE
RavenDB-16139

### PR DESCRIPTION
**Pull request by ppekrol**

### CODE REVIEW

File: src/Voron/Impl/EncryptionBuffersPool.cs [hunk 1]
```diff
@@ -150,11 +146,17 @@ public void Return(byte* ptr, long size, NativeMemory.ThreadStats allocatingThre
             var success = _items[index].TryPush(allocation);
 
             if (success)
+            {
+                // updating the thread allocations since we released the memory back to the pool
+                NativeMemory.UpdateMemoryStatsForThread(allocatingThread, size);
```
Comment (grisha-kotler) [inline src/Voron/Impl/EncryptionBuffersPool.cs:152]: Can we have a test for this?

Comment (ppekrol) [reply src/Voron/Impl/EncryptionBuffersPool.cs:152]: Done. Can you re-check?

Comment (grisha-kotler) [reply src/Voron/Impl/EncryptionBuffersPool.cs:152]: LGTM

File: src/Raven.Server/Config/Categories/IndexingConfiguration.cs [hunk 2]
```diff
@@ -202,7 +202,7 @@ public virtual bool RunInMemory
         public Size? TransactionSizeLimit { get; protected set; }
 
         [Description("Transaction size limit, for encrypted database only, after which an index will stop and complete the current batch")]
-        [DefaultValue(64)]
+        [DefaultValue(96)]
```
Comment (arekpalinski) [inline src/Raven.Server/Config/Categories/IndexingConfiguration.cs:205]: Do we want to backport this to 4.2?

Comment (ppekrol) [reply src/Raven.Server/Config/Categories/IndexingConfiguration.cs:205]: Yes, part of https://github.com/ravendb/ravendb/pull/11681

**Pull request merged**